airflow-core/docs/howto/connection.rst

```diff
@@ -294,7 +294,7 @@ deprecated ``hook-class-names``) in the provider meta-data, you can customize Ai
 * Hiding fields that are not used for your connection
 * Adding placeholders showing examples of how fields should be formatted
 
-You can read more about details how to add custom provider packages in the :doc:`apache-airflow-providers:index`
+You can read more about details how to add custom provider distributions in the :doc:`apache-airflow-providers:index`
```

Comment:
```suggestion
You can read more about details how to add custom providers in the :doc:`apache-airflow-providers:index`
```

Reply:
I changed it all over the places where it make sense.